Mixpanel/Mixpanel.m

```diff
@@ -49,10 +49,33 @@ + (Mixpanel *)sharedInstanceWithToken:(NSString *)apiToken
 
 + (Mixpanel *)sharedInstance
 {
-    if (sharedInstance == nil) {
-        MPLogWarning(@"sharedInstance called before sharedInstanceWithToken:");
+    if (!instances || instances.count == 0) {
```

Comment:
same here can just check `count == 0` because it will be `nil` if `instances` doesn't exist